map/faq.php

```diff
@@ -1,63 +1,15 @@
 <!doctype html>
 <html lang="de-DE">
   <head>
-		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="description" content="Impressum - RailwayStations">
-		<title>FAQ - RailwayStations</title>
-
-    <!-- Bootstrap core CSS -->
-		<link href="bootstrap-4.3.1-dist/css/bootstrap.min.css" rel="stylesheet">
-
-		<link rel="apple-touch-icon" sizes="57x57" href="./images/apple-icon-57x57.png">
-		<link rel="apple-touch-icon" sizes="60x60" href="./images/apple-icon-60x60.png">
-		<link rel="apple-touch-icon" sizes="72x72" href="./images/apple-icon-72x72.png">
-		<link rel="apple-touch-icon" sizes="76x76" href="./images/apple-icon-76x76.png">
-		<link rel="apple-touch-icon" sizes="114x114" href="./images/apple-icon-114x114.png">
-		<link rel="apple-touch-icon" sizes="120x120" href="./images/apple-icon-120x120.png">
-		<link rel="apple-touch-icon" sizes="144x144" href="./images/apple-icon-144x144.png">
-		<link rel="apple-touch-icon" sizes="152x152" href="./images/apple-icon-152x152.png">
-		<link rel="apple-touch-icon" sizes="180x180" href="./images/apple-icon-180x180.png">
-		<link rel="icon" type="image/png" sizes="192x192"  href="./images/android-icon-192x192.png">
-		<link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png">
-		<link rel="icon" type="image/png" sizes="96x96" href="./images/favicon-96x96.png">
-		<link rel="icon" type="image/png" sizes="16x16" href="./images/favicon-16x16.png">
-
-		<link href="css/all.min.css" rel="stylesheet">
-		<link href="css/style.css" rel="stylesheet">
-
+      <title>FAQ - RailwayStations</title>
+      <?php require_once("./header.php") ?>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["require_once" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required.](https://app.codacy.com/app/RailwayStations/RSWebsite/pullRequest?prid=4676723)